src/System.Net.NameResolution/src/System.Net.NameResolution.csproj

```diff
@@ -129,10 +131,10 @@
     <Compile Include="$(CommonPath)\Microsoft\Win32\SafeHandles\SafeLibraryHandle.cs">
```

Comment:
Should this be under `Condition="'$(TargetGroup)' != 'uap'"` as well?

Reply:
Yes, that doesn't break the build.